mono/utils/mono-os-wait-win32-uwp.c

```diff
@@ -9,7 +9,7 @@
 #include <glib.h>
 #include "mono/utils/mono-compiler.h"
 
-#if G_HAVE_API_SUPPORT(HAVE_UWP_WINAPI_SUPPORT)
+#if G_HAVE_API_SUPPORT(HAVE_UWP_WINAPI_SUPPORT | HAVE_XBOXONE_WINAPI_SUPPORT)
```

Comment:
ifdef host_win32 or really I suggest mono ifdef _WIN32 like the rest of the world. It is builtin to the compilers and far more portable.

Reply:
This doesn't include classic win32 desktop, just UWP and Xbox One.

Reply:
Oh. So many ifdefs to riddle the code with , so much. API fragmentation, ok.